jsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "baseUrl": ".",
+        "paths": {
+            "Utils/*": ["./utils/*"],
+            "Services/*": ["./services/*"],
+            "Routes/*": ["./routes/*"],
+            "Config/*": ["./config/*"],
+            "Orm/*": ["./orm/*"],
```

Comment:
Unexpected character ('}' (code 125)): was expecting double-quote to start field name